snap/snapcraft.yaml

```diff
@@ -12,11 +12,16 @@ description: |
 
   Commands can be classified as follows:
 
-    Basic: help, version
-    Action: cut
+    Basic (general operations):
+      find     Find existing slices
+      info     Show information about package slices
+      help     Show help about a command
+      version  Show version details
+
+    Action (make things happen):
+      cut      Cut a tree with selected slices
```

Comment:
```suggestion
      cut      Cut a package with selected slices
```

Reply:
AFAIK "cut a tree with selected slices" is what is currently printed when you run `chisel help --all`.

Reply:
Yup, it's copied from the help. :)